py-polars/polars/dataframe/frame.py

```diff
@@ -5753,9 +5753,11 @@ def drop_in_place(self, name: str) -> Series:
 
     def clear(self, n: int = 0) -> Self:
         """
-        Create an empty copy of the current DataFrame, with zero to 'n' rows.
+        Create an empty copy of the current DataFrame, with `n` rows.
 
-        Returns a DataFrame with identical schema but no data.
+        Returns a DataFrame with an identical schema, but no data (i.e., all
+        values in every column in the copy are set to null).
+        `n` can be greater than the current number of rows in the DataFrame.
```

Comment:
How about:

> Returns a null-filled DataFrame with an identical schema.

Reply:
That is fine, but mine explains more clearly to someone who is not familiar with what exactly happens, if you've never used it.

Reply:
You are saying "but no data", which can either mean no rows, or all nulls. I think @mcrumiller  version is more exact.

Reply:
The "no data" part was what was originally in the docstring. I agree that it's vague. 

Final proposal:
```
Returns a null-filled DataFrame with an identical schema.
`n` can be greater than the current number of rows in the DataFrame.
```